core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -817,6 +817,18 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val FILE_INSERT_STATUS_LOG_FLAG =
+    ConfigBuilder("spark.file.insert.status.log.flag")
```

Comment:
Do we have another configuration in `spark.file.` namespace? If not, we usually do not make a new namespace.